homeassistant/components/mqtt/mixins.py

```diff
@@ -206,6 +226,10 @@ def attributes_message_received(msg: Message) -> None:
                     payload = attr_tpl.async_render_with_possible_json_value(payload)
                 json_dict = json.loads(payload)
                 if isinstance(json_dict, dict):
+                    for k in MQTT_ATTRIBUTES_BANNED:
+                        json_dict.pop(k, None)
```

Comment:
I would expect `json_dict` to be smaller than the `MQTT_ATTRIBUTES_BANNED` and so making a new dict with keys not in the banned attributes might be faster? 🤷